models/task.py

```diff
@@ -1,13 +1,13 @@
 from models.configuration import Configuration
 from models.notification import Notification
-from models.schedule import Schedule
+from models.schedule.factory import Factory
 
 
 class Task:
     def __init__(self, json_task):
         self.name = json_task["name"]
         self.kind = json_task["kind"]
-        self.schedule = Schedule(json_task["schedule"])
+        self.schedule = Factory(json_task["schedule"])
```

Comment:
```suggestion
        self.schedule = Factory(json_task["schedule"]).new()
```

Ai aqui a mudaça seria só isso

Reply:
Considerado no ajuste do factory